README.md

```diff
@@ -34,8 +34,6 @@ enter the URL for your definition and select **TRY IT**.
 
 - Responsive three-panel design with menu/scrolling synchronization
 - Support for OpenAPI 3.1, OpenAPI 3.0, and Swagger 2.0
-- [Multiple deployment options](https://redocly.com/docs/redoc/)
```

Comment:
Why is this removed?

Reply:
It's repeated in the next section and I think the in-page examples make more sense in a README context.